maven/viatra-query-language/pom.xml

```diff
@@ -37,6 +37,11 @@
             <version>${xtend.compiler.version}</version>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>org.eclipse.emf</groupId>
+            <artifactId>org.eclipse.emf.codegen.ecore</artifactId>
+			<!-- Version locked down by xtext-dev-bom in org.eclipse.viatra.parent -->
```

Comment:
Can you fix the indentation please? Thanks.

Reply:
Done.